rfcs/NNNN-lit-3.0.md

```diff
@@ -0,0 +1,112 @@
+---
+Status: Active
+Champions: @justinfagnani
+PR: https://github.com/lit/rfcs/pull/16
+---
+
+# Lit 3.0
+
+A proposal for the next major release of Lit
+
+## Objective
+
+Allow us to make a few breaking changes that will let us reduce some engineering complexity, and very lightly reduce bundle size.
+
+### Goals
```

Comment:
Is there a way we can somehow integrate template hydration into the main library without a performance hit? Load order of hydration helpers is DX pain and the double rendering of templates is a very common symptom

Reply:
I don't think so. There's just extra code that CSR doesn't need.